pkg/util/util.go

```diff
@@ -8,6 +8,8 @@ const ExtensionAPIServerAuthenticationConfigMap = "extension-apiserver-authentic
 const RequestHeaderClientCAFileKey = "requestheader-client-ca-file"
 const VirtShareDir = "/var/run/kubevirt"
 const VirtLibDir = "/var/lib/kubevirt"
+const CpuManagerOpenShift3Path = "/var/lib/origin/openshift.local.volumes/cpu_manager_state"
```

Comment:
Will the OKD 4 also have a different path?

Reply:
yes

Reply:
Can you please add it under the PR?

Reply:
I will :) working on that :)

Reply:
I found only one path `/sysroot/ostree/deploy/rhcos/var/lib/kubelet/cpu_manager_state`

Reply:
and I think it is correct one

Reply:
ah yes, I meant it's different than the path in os-3.11 :). Btw. funny thing... :D I am going to drop all changes here, after speaking with @fabiand and @vladikr I am going to leave the old solution with cpu-manager on os-3.11 and for okd4.1 I will try to get the information about policy from node configmap. The reason why is we don't want to add more hostPath(s) to virt-handler.

Reply:
I see you will need to check `KubeletConfigs`, for me, it looks pretty complicated because it can have a number of `KubeletConfigs` and each config can be only valid for specific nodes :(

Reply:
maybe `virt-handler` could check `KubeletConfig` for its node, it sounds right. Do you see any risk here?

Reply:
I can imagine flow like this:

1. virt-handler checks on which node is running (maybe that info is available already)
2. virt-handler checks node annotation `machineconfiguration.openshift.io/currentConfig`
3. virt-handler gets `machineconfiguration` based on the value from step 2, I see there is the content of kubelet.conf

that should work.